py/bitbox02/bitbox02/util.py

```diff
@@ -141,3 +144,68 @@ def set_app_static_privkey(self, privkey: bytes) -> None:
         data = self._read_cache()
         data.app_static_privkey = privkey
         self._write_cache(data)
+
+
+class BitBoxAppNoiseConfig(bitbox_api_protocol.BitBoxNoiseConfig):
+    """
+    Noise config that reads and stores the noise keys in the same location as the BitBoxApp.
+    """
+
+    def __init__(self) -> None:
+        system = platform.system()
+        if system == "Linux":
+            folder = Path(
+                os.environ.get(
+                    "XDG_CONFIG_HOME",
+                    # fallback
+                    os.path.join(os.environ["HOME"], ".config"),
+                )
+            )
+        elif system == "Darwin":
+            folder = Path(os.environ["HOME"]) / "Library" / "Application Support"
+        elif system == "Windows":
+            folder = Path(os.environ["APPDATA"])
+        else:
+            raise NotImplementedError("Unknown system: {}".format(system))
+        self._filename = folder / "bitbox" / "bitbox02" / "bitbox02.json"
+
+    def _read_cache(self) -> Any:
```

Comment:
Not really a cache, is it. How about `_read_config` and `_write_config`?

Reply:
Done (just _read and _write)